actionpack/test/controller/http_basic_authentication_test.rb

```diff
@@ -170,6 +170,12 @@ def test_encode_credentials_has_no_newline
     assert_response :unauthorized
   end
 
+  test "authentication request with wrong credentials" do
```

Comment:
We could make it more explicit what is wrong with the credentials.
```suggestion
  test "authentication request with credentials without colon" do
```
or something like...
```suggestion
  test "authentication request with credentials missing colon" do
```

Reply:
I agree with you. 
I think the first suggestion is more suitable. Applied it and squashed commits.
Thanks for the review!